app/models/green_flag/feature.rb

```diff
@@ -61,7 +63,17 @@ def fully_enabled?
   def fully_disabled?
     rules.count == 0 || rules.all? { |rule| rule.percentage == 0 }
   end
-  
+
+  def requires_manual_deletion?
+    feature_decisions.count >= DELETION_MAX_FEATURE_DECISIONS
```

Comment:
For a really large number of feature_decisions, just counting them up can take 30+ seconds.  I'm thinking we can just ignore the issue - if your request times out, then you'll need to intervene manually.